phc/easy/dstu3.py

```diff
@@ -57,9 +57,9 @@ def put(self, record_id: str, data: dict, auth_args: Auth = Auth.shared()):
 
         response = client._fhir_call(
             f"{self.entity}/{record_id}", http_verb="PUT", json=data
-        ).data
+        )
 
-        if response == "OK":
+        if response.status_code == 200 or response.status_code == 201:
```

Comment:
Maybe this should just be:

```suggestion
        if 200 <= response.status_code < 300:
```

Reply:
I was trying to keep it strict with the FHIR RESTful API standard, but I agree. It should be fine if it's more lenient.

Reply:
Ah, yeah I'm OK with being strict to the standard. I was just thinking to avoid needing to come adjust this code in the future.